src/uct/base/uct_md.c

```diff
@@ -50,6 +50,12 @@ ucs_config_field_t uct_md_config_rcache_table[] = {
      "Maximal total size of registration cache regions",
      ucs_offsetof(uct_md_rcache_config_t, max_size), UCS_CONFIG_TYPE_MEMUNITS},
 
+    {"RCACHE_CLEANUP_THRESH", "4G",
```

Comment:
Need to find a meaningful value here, I picked this quite randomly.

Reply:
let's make it 512M